Droidcon-Boston/app/build.gradle

```diff
@@ -52,6 +52,8 @@ dependencies {
   implementation "com.android.support:cardview-v7:$support"
   implementation "com.android.support:design:$support"
 
+  // Firebase cloud messaging
+  implementation "com.google.firebase:firebase-messaging:$playServices"
```

Comment:
Doesn't this file need

apply plugin: 'com.google.gms.google-services'

at the end of it?  I'm trying to add Firebase DB and I think it needs this...

Reply:
ugh....google_service.json is also in the wrong directory...it should be in the app directory :-P
I can move it my PR if you want or move it in yours if you need it.
Reading JSON in Java looks like it'll be a PITA :-(

Reply:
Does this pull request have a DB? It's all GCM and notifications. Looks like @kenyee can push a branch that will fix the gradle with Proguard to drop Log.d and add in the additional play services for the FB Real-Time Database.

Reply:
@kenyee - if you don't like pita then export to a csv and reuse the sqlite code from last year for the local data.

Reply:
I missed the google-services plugin at the bottom. Was my new PR after the discussion to close the last one. 
I fixed it, tested the app for notification, updated the proguard-rules, moved the strings to strings.xml.
About adding realtime db, I was thinking others were working on it and would add it in their own PR. Its not added here